RELEASENOTES.md

```diff
@@ -11,6 +11,20 @@ If you are using Red Hat Enterprise Linux 7 or CentOS 7, you can [update using `
 
 # Releases
 
+## Version 0.81.0
+
+The `rippled` 0.81.0 release introduces changes that improve the scalability of the XRP Ledger and transitions the recommended validator configuration to a new hosted site, as described in Ripple's [Decentralization Strategy Update](https://ripple.com/dev-blog/decentralization-strategy-update/) post.
+
+**New and Updated Features**
+
+- New hosted validator configuration.
+
+
+**Bug Fixes**
+
+- Filter SQL results in lieu of a search clause ([#2312](https://github.com/ripple/rippled/isses/2312))
```

Comment:
The URL leads to a 404.  Is "isses" really supposed to be part of the path?

Reply:
The description isn't accurate anymore either. The updated commit message is better: `Optimize queries for account_tx to work around SQLite query planner`. Or simplified as `Optimize long-running SQL queries`

Reply:
isses --> issues